dolphin/drivers/api.py

```diff
@@ -78,8 +78,9 @@ def remove_trap_config(self, context, storage_id, trap_config):
 
     def parse_alert(self, context, storage_id, alert):
         """Parse alert data got from snmp trap server."""
-        driver = self.driver_manager.get_driver(context, **{'storage_id': storage_id})
-        driver.parse_alert(context, alert)
+        driver = self.driver_manager.get_driver(context,
+                                                **{'storage_id': storage_id})
```

Comment:
is it better to use storage_id=storage_id?

Reply:
Done